properties-files/openshiftv4_on_aws.properties

```diff
@@ -12,6 +12,23 @@ SCALE_CI_BUILD_TRIGGER_URL=
 OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=
 OPENSHIFT_INSTALL_BINARY_URL=
 ENABLE_DITTYBOPPER=true
+CERBERUS_ENABLE=true
+KUBECONFIG_PATH=/root/.kube/config
+CERBERUS_CONFIG_PATH=/root/cerberus.yml
+CERBERUS_IMAGE=quay.io/openshift-scale/cerberus:latest
+CERBERUS_URL=http://0.0.0.0:8080
+WATCH_NODES=true
+WATCH_NAMESPACES=[openshift-etcd, openshift-apiserver, openshift-kube-apiserver, openshift-monitoring, openshift-kube-controller, openshift-machine-api, openshift-kube-scheduler, openshift-ingress, openshift-sdn, openshift-ovn-kubernetes]
```

Comment:
"openshift-sdn" will take care of monitoring either openshift-sdn ns or openshift-ovn-kubernetes ns

Reply:
Fixed it, thanks.